### ISSUE TITLE
Progress event for WebAssembly compilation?

### ISSUE THREAD

**Issue by Auron52** [WebAssembly/design#1135]
Unless I am missing something there is currently no way to get progress events for WebAssembly compilation/instantiation in the browser. This is especially relevant on mobile devices where compilation is potentially very slow. Is this something that has been considered or could be considered for a future extension to WebAssembly?

**Comment by jfbastien** [comment]
That is correct. Maybe precedents with fetch could guide what progress events should look like? @domenic / @slightlyoff may have opinions about this.
